src/core/src/sim/mc/moves/rotate.rs

```diff
@@ -20,41 +22,38 @@ pub struct Rotate {
     molid: usize,
     /// New positions of the atom in the rotated molecule
     newpos: Vec<Vector3D>,
-    /// Normal distribution, for generation of the axis
-    axis_rng: Normal,
-    /// Range distribution, for generation of the angle
-    angle_rng: Range<f64>,
+    /// Displacement
+    displacement: Displacement,
 }
 
 impl Rotate {
     /// Create a new `Rotate` move, with maximum angular displacement of `theta`,
     /// rotating all the molecules in the system.
-    pub fn new(theta: f64) -> Rotate {
-        Rotate::create(theta, None)
+    pub fn new(theta: f64, update_freq: u64, target_acceptance: f64) -> Rotate {
```

Comment:
I would not add all the parameters to the constructor, but rather use the default and have setters when deviating from the default.

Something like this:

```rust
pub fn new(theta: f64) -> Rotate {
      // same as before, using Displacement::default for the initial value of displacement
}

pub fn set_target_acceptance(&mut self, target: f64) { /* ... */ }

pub fn set_update_frequency(&mut self, frequency: f64) { /* ... */ }
```